src/sentry/monitors/logic/mark_failed.py

```diff
@@ -30,6 +30,10 @@ def mark_failed(
     if monitor_env is None:
         return False
 
+    # Use the failure time as recieved if there is no received time
+    if received is None:
+        received = failed_at
```

Comment:
This is just a default and isn't actually used anywhere in the code except for tests.